webui/react/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "engines": {
     "node": ">=16.13 <17",
-    "npm": ">=8.0.0"
+    "npm": ">=9.0.0"
```

Comment:
Constraint npm version

Reply:
@ashtonG @keita-determined do we want constraint it further related to the recent Vite changes?
> If you’re using npm with a version between 9.0.0  and 9.4.1, please update to the latest version of npm. npm 9 introduced a backwards-incompatible change to how local dependencies work by default which was recently fixed.

Reply:
oh nevermind, I see the newer updates to be newer than `9.4.2`